database/redis/database.go

```diff
@@ -203,3 +196,22 @@ func (connector *DbConnector) Client() redis.UniversalClient {
 func (connector *DbConnector) Context() context.Context {
 	return connector.context
 }
+
+// callDependent calls the fn dependent of Redis client type (cluster or standalone).
+func (connector *DbConnector) callDependent(fn func(connector *DbConnector, client redis.UniversalClient) error) error {
```

Comment:
maybe named better: callFunc? 

verb + adjective seems not good :)

Reply:
and can we write a test on this small func?

Reply:
renamed here https://github.com/moira-alert/moira/pull/778/commits/1f59f98dd5964a1ec452cf468e0385620912e3f1.

This func is internal, so it is already (partially) covered by tests and tests are passed. I was thinking about test, and we can test that function `fn` was called once or 3 times, and that test will depend on current Redis client type, cluster or standalone. I think unit test should not depend on current Redis client type (it should be private details). And it is problem that we cannot fully test this function, it is covered only by half of it.